spring-cloud-data-rest-resource/src/main/java/org/springframework/cloud/data/rest/resource/TaskDefinitionResource.java

```diff
@@ -30,6 +31,12 @@
 
 	private String dslText;
 
+	/**
+	 * Stream status (i.e. running, complete, etc).
+	 */
+	private String status = "";
```

Comment:
So here is the question.  What is the state of a task that has not been run?  I assumed it was blank or we create a State of "Ready".  Then when run, then it would change to the appropriate state.